### PR TITLE
Web Scrapper

### PR DESCRIPTION
**Pull request by darielbobadilla**
This program reads the University xml schedule and creates a cvs file.

### CODE REVIEW

File: Web_Scraper_Schedule.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+				Argument = 'NA'
+			try:
+				Operator = Requirement['operator']
+			except:
+				Operator = 'NA'
+			try:
+				value = Requirement["value2"]
+			except:
+				value = 'NA'
+			data.append([title,section,call_number,MinCredit,MaxCredit,MaxEnrollment,Cur_Enroll,Status,StartDate,EndDate,Instructor,Term,Meeting_day,start_time,end_time,site,Building,Room,Activity,control,Argument,value,Operator])
```
Comment (CCorrado) [inline Web_Scraper_Schedule.py]: After this for loop is done executing, you should be able to take 2 copies of `data` and pass it into record linkage.

https://recordlinkage.readthedocs.io/en/stable/ref-index.html
https://pypi.org/project/recordlinkage/

Comment (CCorrado) [reply Web_Scraper_Schedule.py]: ( you can do this in a separate branch ) but having data as a Data Frame is a good start

File: Web_Scraper_Schedule.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+				Room = meeting['room']
+			except:
+				Room = 'NA'
+			try:
+				Activity = meeting['activity']
+			except:
+				Activity = 'NA'
+			try:
+				Requirement = course.find('requirement')
+				control = Requirement['conrol']
```
Comment (CCorrado) [inline Web_Scraper_Schedule.py]: is `control` misspelled in the data as `conrol`?

File: Web_Scraper_Schedule.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+#pip install requests 
+from bs4 import BeautifulSoup
+#pip install beautifulsoup4
+import csv
+
+def main():
+	page = requests.get("https://web.stevens.edu/scheduler/core/core.php?cmd=getxml&term=2019F")
+	# print(page.content)
+	soup = BeautifulSoup(page.text,'html.parser')
+	with open("Term_Schedule.csv",'a') as csvFile:
```
[the remark this answers is not in the document]
Comment (darielbobadilla) [reply Web_Scraper_Schedule.py]: Done

[the remark this answers is not in the document]
Comment (CCorrado) [reply Web_Scraper_Schedule.py]: I meant `Term_Schedule.csv` -- still can't run the script without it

**Pull request merged**